src/clients/java/src/jni_tests.zig

```diff
@@ -1369,21 +1443,63 @@ test "JNI: primitive arrays" {
                     };
                 }
 
-                fn release_array_elements(env: *jni.JNIEnv, array: jni.JArray, elements: [*]PrimitiveType) void {
+                fn release_array_elements(
```

Comment:
This actually looks like it could benefit from some larged indentation-removing refator (like pulling `ArrayTestType` outside of `test "JNI: primitive arrays"`), but probably doesn't matter much for tests. Can adjust the next time we touch this!